scripts/build-android.sh

```diff
@@ -72,5 +72,7 @@ else
   nixOpts+=("--option" "build-use-sandbox" "true")
 fi
 
+# needed since react-native v0.75
+source "${GIT_ROOT}/scripts/generate_autolink_android.sh"
```

Comment:
Why are you sourcing it if you don't use it?

Reply:
Oh my bad this maybe confusing, The thing is that `generate_autolink_android.sh` has a function named `generate_android_autolink()` and that function is called at the end of the script. 
So sourcing it also ensures that the required function is executed.